src/Telemetry/Data_Providers/Debug_Data.php

```diff
@@ -46,9 +47,34 @@ public function get_data(): array {
 				unset( $plugins[ $active_plugin ] );
 			}
 		}
+
 		$info['telemetry-active-plugins']['fields']   = $active;
 		$info['telemetry-inactive-plugins']['fields'] = $plugins;
 
 		return $info;
 	}
+
+	/**
+	 * Some data in Site Health is marked as private.
+	 * This ensures we don't pass that on to the servers.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,mixed> $data Raw Site Health data.
+	 *
+	 * @return array<string,mixed> Filtered Site Health data.
+	 */
+	public function clean_private_data( $data ): array {
```

Comment:
If you're **sure** the `$data` parameter will always be an array, then type-hint it.
If you cannot be always sure of that, add an `is_array($data)` check to bail if `$data` is not an array.

Reply:
Same applies to the return type: either you're sure of the type or not.
If you're not, remove the return type.